SI/Sem.03/Pract.03/Таsк05.cpp

```diff
@@ -0,0 +1,22 @@
+#include <iostream>
+int main() {
+	int number = 1, testNum = 0, count = 0;
+	std::cout << "Input number between 1 and 100: ";
+	std::cin >> number;
+	if (number >= 1 && number <= 100) {
+		do {
+			std::cin >> testNum;
+			if (testNum > number) {
+				std::cout << testNum << " > " << std::endl;
+			}
+			if (testNum < number) {
```

Comment:
If the first condition is true, you do not need to check the second one too. Use _else if_.